tests/e2e/vLLM/configs/w4a16_grouped_quant_qwen.yaml

```diff
@@ -0,0 +1,7 @@
+cadence: "nightly"
+test_type: "regression"
+model: Qwen/Qwen2.5-0.5B
+scheme: W4A16
```

Comment:
Maybe we can swap this one with W8A8 int8? we have good W4A6 coverage with what you've added.